src/Path/Module.hs

```diff
@@ -19,38 +19,38 @@ import Path.Pretty
 import Text.PrettyPrint.ANSI.Leijen hiding ((<$>))
 import Text.Trifecta.Rendering (Span)
 
-data Module v a ann = Module
+data Module v a = Module
   { moduleName    :: ModuleName
   , moduleDocs    :: Maybe String
   , modulePath    :: FilePath
-  , moduleImports :: [Import ann]
+  , moduleImports :: [Import]
   , moduleDecls   :: [Decl v a]
   }
-  deriving (Eq, Ord, Show)
+  deriving (Eq, Foldable, Functor, Ord, Show, Traversable)
 
-data Import ann = Import { importModuleName :: ModuleName, importAnn :: ann }
+data Import = Import { importModuleName :: ModuleName, importAnn :: Span }
```

Comment:
I specialized `Import` to `Span` annotations because there’s not really any need for any other kind of annotation.